packages/elements/src/components/table/table.stories.mdx

```diff
@@ -8,9 +8,16 @@ import {
   TableCell,
   TableExpandableRowTriggerCell,
   TableExpandableRow,
+  TableRowContainer,
 } from './molecules'
 import { Icon } from '../icon'
 import { RenderHtmlMarkup } from '../../helpers/render-html-markup'
+import { Grid, Col, elSpan6, elOffset7 } from '../grid'
+import { Molecule } from '../layout'
+import { Subtitle } from '../typography'
+import { InputGroup } from '../input-group'
+import { Button } from '../button'
+import { StatusIndicator } from '../status-indicator'
```

Comment:
A general comment but an `<ArgsTable component={Table} story="Foobar" />` component would be good on this page - one for each of the table cell and table components - just helps with understanding the available props

Reply:
sure :)

Reply:
I've added some but if you add an args table for components with no uniquely defined props `export const Example = () => ()` then you get some odd results like the previous component's props will appear in the args table